react-data-converter/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Client Ignores
```

Comment:
Why do we have a .gitignore inside the global .gitignore? Can we migrate this into the global .gitignore?